### ISSUE TITLE
Social media links are not accessible via a screen reader

### ISSUE THREAD

**Issue by gregdorward** [redbadger/react.london#326]
**Description of the issue:**
Currently, a user cannot determine the content of the social media links in the footer of the React London pages
**Browser used:**
Any with a screen reader
**OS used:**
All
**Device used:**
Desktop

**Steps to reproduce:**

1. View the HTML code for any of the react pages
2. Inspect any of the social media links in the footer

**Actual result:**
To a screen reader, these links are non-discernible

**Expected result:**
The links should be tagged so that a screen reader can inform the user of their purpose
